Phraseanet-production-client/src/components/record/pushOrShare/pushOrShare.js

```diff
@@ -130,6 +130,44 @@ const pushOrShare = function (services, container) {
         return false;
     });
 
+    $('.Sharebasket .light_button').hover(function() {
+            var that = $(this);
+            if($("INPUT[name=isFeedback]").val() === '0') {
+                // just a share
+                that.css({'background-color':'#4caf50', 'color': '#fff'});
+            } else {
+                that.css({'background-color': '#E0215D', 'color': '#fff'});
+            }
+        }, function () {
+            var that = $(this);
+            if($("INPUT[name=isFeedback]").val() === '0') {
+                // just a share
+                that.css({'background-color':'#fff', 'color': '#4caf50'});
+            } else {
+                that.css({'background-color': '#fff', 'color': '#E0215D'});
+            }
+        }
+    );
+
```

Comment:
no hardcoded colors in .js, only in .scss ; use color aliases, harcode theme color only once (except for "standard" colors like black/white).
in js, add-or-remove-or-change a class.